capa/main.py

```diff
@@ -933,6 +935,11 @@ def main(argv=None):
             logger.error("Input file '%s' is not a valid PE file: %s", args.sample, str(e))
             return E_CORRUPT_FILE
 
+        is_dotnet = is_dotnet_file(file_extractor.pe)
+        if is_dotnet:
+            # TODO try/except
+            file_extractor = capa.features.extractors.dotnetfile.DotnetfileFeatureExtractor(args.sample)
```

Comment:
#938: eventually probably want to collect a list of extractors to run so we can handle both native + .NET

i understand this logic will undergo further revisions, just hyperlinking issues